drkafka/src/main/java/com/pinterest/doctorkafka/config/DoctorKafkaConfig.java

```diff
@@ -18,6 +18,7 @@
 
 public class DoctorKafkaConfig {
 
+  private static final String OLDKAFKA = "oldkafka";
```

Comment:
as kafka 0.10 has been releases for about two years, and most of kafka users have been on kafka 0.10 or newer version. shall we just update DoctorKafka to use the latest api?

Reply:
It might be good to keep the support on for some time and mark this as deprecated since we don't have version based branches.

Reply:
then it will require another change to remove the deprecated code. 

if we would like to keep the support,  can we be more clear on the versions that we consider as "oldKafka"?  the name "oldKafka" can be confusing to the readers.  it will be good to clarify that.